app/javascript/app/components/table/table.js

```diff
@@ -35,6 +35,23 @@ class TableContainer extends PureComponent {
     }
   }
 
+  setRowsHeight = columns => {
+    if (columns.find(c => c.value === 'description')) {
+      return 150;
+    }
+    return 50;
+  };
+
+  setColumnWidth = column => {
+    if (column === 'year') {
+      return 80;
+    }
+    if (column === 'description') {
```

Comment:
Could you add 'description' to the fullTextColumns in models in the selectors file. As we are displaying it also there it might get some more height instead of the ellipsis